Request to merge branches

reveillette opened the pull request:
Hi Patrick, I figured out how to merge your gh-pages branch with mine—through pull requests.

leidio committed:
Added How-to Page

Including Joyride script for how-to page guide tour

leidio committed:
further how-to page revisions

Added Museo, completed build-out of how-to page sections. Awaiting copy
from client.

leidio committed:
Added Simptip for tour

Removed Joyride javascript and implemented a hover tooltip for the tour.

leidio committed:
Style updates to Index, Library

leidio committed:
Style changes

leidio committed:
Header changes

leidio committed:
Nav bar style

leidio committed:
Style

I know it’s annoying that I keep committing little style changes but
it’s the only way I can verify that the changes I’m making in Chrome
are executed.

leidio committed:
Repository Commit

Recommitting the entire repository after a clusterfuck session with
Google Drive.

leidio committed:
Style changes

leidio committed:
Style

leidio committed:
shitloads of updates to how to

leidio committed:
style changes

leidio committed:
style changes

leidio committed:
more style

sorry, trying to see my changes take effect

leidio committed:
nav links

leidio committed:
nav style

leidio committed:
Universal typography

Everything is now Source Sans Pro which makes the whole site look much
cleaner and less cheesy!

leidio committed:
javascript page navigation

leidio committed:
Style

leidio committed:
Initial Bootstrap deployment

This compilation of Bootstrap only uses pagination and its grid
architecture. Everything else was left off. Can be changed, of course.

leidio committed:
FULL Bootstrap deploy

Gave up trying to do a custom deployment, which would have made
responsiveness unstable

leidio committed:
Bootstrap fixes

Bootstrap now performs on index.html and howto.html

leidio commented:
ah! just seeing this.

I'm not really sure what to do... the merge request on github says that it can't automatically merge them, and gives some command line instruction that didn't work.

thoughts?

*P A T R I C K http://patrick.thedotank.org    *

On Thu, Mar 12, 2015 at 3:01 PM, Andrea Hansen notifications@github.com
wrote:

> Hi Patrick, I figured out how to merge your gh-pages branch with
> 
> ## mine—through pull requests.
> 
> You can view, comment on, or merge this pull request online at:
> 
>   https://github.com/reveillette/DCDC-website/pull/1
> Commit Summary
> - Added How-to Page
> - further how-to page revisions
> - Added Simptip for tour
> - Style updates to Index, Library
> - Style changes
> - Header changes
> - Nav bar style
> - Style
> - Repository Commit
> - Style changes
> - Style
> 
> File Changes
> - _D_ DCDC-website.sublime-project
>   https://github.com/reveillette/DCDC-website/pull/1/files#diff-0 (9)
> - _D_ DCDC-website.sublime-workspace
>   https://github.com/reveillette/DCDC-website/pull/1/files#diff-1
>   (751)
> - _A_ assets/alien.svg
>   https://github.com/reveillette/DCDC-website/pull/1/files#diff-2
>   (159)
> - _A_ assets/books.svg
>   https://github.com/reveillette/DCDC-website/pull/1/files#diff-3
>   (278)
> - _A_ assets/detroit.jpg
>   https://github.com/reveillette/DCDC-website/pull/1/files#diff-4 (0)
> - _A_ assets/flag.svg
>   https://github.com/reveillette/DCDC-website/pull/1/files#diff-5
>   (113)
> - _A_ assets/hipster.jpg
>   https://github.com/reveillette/DCDC-website/pull/1/files#diff-6 (0)
> - _A_ assets/howtofull.png
>   https://github.com/reveillette/DCDC-website/pull/1/files#diff-7 (0)
> - _A_ assets/icons/wrench.svg
>   https://github.com/reveillette/DCDC-website/pull/1/files#diff-8 (22)
> - _A_ assets/kids.jpg
>   https://github.com/reveillette/DCDC-website/pull/1/files#diff-9 (0)
> - _A_ assets/music.svg
>   https://github.com/reveillette/DCDC-website/pull/1/files#diff-10
>   (105)
> - _A_ assets/network.svg
>   https://github.com/reveillette/DCDC-website/pull/1/files#diff-11
>   (132)
> - _A_ assets/placemarker.svg
>   https://github.com/reveillette/DCDC-website/pull/1/files#diff-12
>   (74)
> - _A_ assets/ribbon.svg
>   https://github.com/reveillette/DCDC-website/pull/1/files#diff-13
>   (12)
> - _A_ assets/science.png
>   https://github.com/reveillette/DCDC-website/pull/1/files#diff-14 (0)
> - _A_ assets/star.png
>   https://github.com/reveillette/DCDC-website/pull/1/files#diff-15 (0)
> - _A_ assets/star.svg
>   https://github.com/reveillette/DCDC-website/pull/1/files#diff-16
>   (16)
> - _A_ assets/table.jpeg
>   https://github.com/reveillette/DCDC-website/pull/1/files#diff-17 (0)
> - _A_ css/joyride.css
>   https://github.com/reveillette/DCDC-website/pull/1/files#diff-18
>   (262)
> - _M_ css/library.css
>   https://github.com/reveillette/DCDC-website/pull/1/files#diff-19
>   (27)
> - _A_ css/museo/MuseoSans-100-demo.html
>   https://github.com/reveillette/DCDC-website/pull/1/files#diff-20
>   (611)
> - _A_ css/museo/MuseoSans-100-webfont.eot
>   https://github.com/reveillette/DCDC-website/pull/1/files#diff-21 (0)
> - _A_ css/museo/MuseoSans-100-webfont.svg
>   https://github.com/reveillette/DCDC-website/pull/1/files#diff-22
>   (815)
> - _A_ css/museo/MuseoSans-100-webfont.ttf
>   https://github.com/reveillette/DCDC-website/pull/1/files#diff-23 (0)
> - _A_ css/museo/MuseoSans-100-webfont.woff
>   https://github.com/reveillette/DCDC-website/pull/1/files#diff-24 (0)
> - _A_ css/museo/MuseoSans-100-webfont.woff2
>   https://github.com/reveillette/DCDC-website/pull/1/files#diff-25 (0)
> - _A_ css/museo/MuseoSans-300-demo.html
>   https://github.com/reveillette/DCDC-website/pull/1/files#diff-26
>   (611)
> - _A_ css/museo/MuseoSans-300-webfont.eot
>   https://github.com/reveillette/DCDC-website/pull/1/files#diff-27 (0)
> - _A_ css/museo/MuseoSans-300-webfont.svg
>   https://github.com/reveillette/DCDC-website/pull/1/files#diff-28
>   (860)
> - _A_ css/museo/MuseoSans-300-webfont.ttf
>   https://github.com/reveillette/DCDC-website/pull/1/files#diff-29 (0)
> - _A_ css/museo/MuseoSans-300-webfont.woff
>   https://github.com/reveillette/DCDC-website/pull/1/files#diff-30 (0)
> - _A_ css/museo/MuseoSans-300-webfont.woff2
>   https://github.com/reveillette/DCDC-website/pull/1/files#diff-31 (0)
> - _A_ css/museo/MuseoSans_700-demo.html
>   https://github.com/reveillette/DCDC-website/pull/1/files#diff-32
>   (611)
> - _A_ css/museo/MuseoSans_700-webfont.eot
>   https://github.com/reveillette/DCDC-website/pull/1/files#diff-33 (0)
> - _A_ css/museo/MuseoSans_700-webfont.svg
>   https://github.com/reveillette/DCDC-website/pull/1/files#diff-34
>   (859)
> - _A_ css/museo/MuseoSans_700-webfont.ttf
>   https://github.com/reveillette/DCDC-website/pull/1/files#diff-35 (0)
> - _A_ css/museo/MuseoSans_700-webfont.woff
>   https://github.com/reveillette/DCDC-website/pull/1/files#diff-36 (0)
> - _A_ css/museo/MuseoSans_700-webfont.woff2
>   https://github.com/reveillette/DCDC-website/pull/1/files#diff-37 (0)
> - _A_ css/museo/MuseoSans_900-demo.html
>   https://github.com/reveillette/DCDC-website/pull/1/files#diff-38
>   (611)
> - _A_ css/museo/MuseoSans_900-webfont.eot
>   https://github.com/reveillette/DCDC-website/pull/1/files#diff-39 (0)
> - _A_ css/museo/MuseoSans_900-webfont.svg
>   https://github.com/reveillette/DCDC-website/pull/1/files#diff-40
>   (819)
> - _A_ css/museo/MuseoSans_900-webfont.ttf
>   https://github.com/reveillette/DCDC-website/pull/1/files#diff-41 (0)
> - _A_ css/museo/MuseoSans_900-webfont.woff
>   https://github.com/reveillette/DCDC-website/pull/1/files#diff-42 (0)
> - _A_ css/museo/MuseoSans_900-webfont.woff2
>   https://github.com/reveillette/DCDC-website/pull/1/files#diff-43 (0)
> - _A_ css/museo/generator_config.txt
>   https://github.com/reveillette/DCDC-website/pull/1/files#diff-44 (5)
> - _A_ css/museo/museo.css
>   https://github.com/reveillette/DCDC-website/pull/1/files#diff-45
>   (64)
> - _A_ css/simptip.css
>   https://github.com/reveillette/DCDC-website/pull/1/files#diff-46
>   (447)
> - _M_ css/styles.css
>   https://github.com/reveillette/DCDC-website/pull/1/files#diff-47
>   (281)
> - _A_ howto.html
>   https://github.com/reveillette/DCDC-website/pull/1/files#diff-48
>   (250)
> - _M_ html/header.html
>   https://github.com/reveillette/DCDC-website/pull/1/files#diff-49 (6)
> - _M_ index.html
>   https://github.com/reveillette/DCDC-website/pull/1/files#diff-50
>   (64)
> - _A_ js/jquery-1.10.1.js
>   https://github.com/reveillette/DCDC-website/pull/1/files#diff-51
>   (9807)
> - _A_ js/jquery.cookie.js
>   https://github.com/reveillette/DCDC-website/pull/1/files#diff-52
>   (72)
> - _A_ js/jquery.joyride-2.1.js
>   https://github.com/reveillette/DCDC-website/pull/1/files#diff-53
>   (928)
> - _A_ js/modernizr.mq.js
>   https://github.com/reveillette/DCDC-website/pull/1/files#diff-54
>   (224)
> - _T_ js/svg.draggable.js
>   https://github.com/reveillette/DCDC-website/pull/1/files#diff-55 (0)
> - _T_ js/svg.draggable.min.js
>   https://github.com/reveillette/DCDC-website/pull/1/files#diff-56 (0)
> - _M_ library.html
>   https://github.com/reveillette/DCDC-website/pull/1/files#diff-57
>   (14)
> 
> Patch Links:
> - https://github.com/reveillette/DCDC-website/pull/1.patch
> - https://github.com/reveillette/DCDC-website/pull/1.diff
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/reveillette/DCDC-website/pull/1.

leidio committed:
Share form adjustments

leidio committed:
margin on social buttons

leidio commented:
So what should i do now?

​

*P A T R I C K http://patrick.thedotank.org    *

On Tue, Mar 31, 2015 at 2:57 PM, Andrea Hansen notifications@github.com
wrote:

> Reopened #1 https://github.com/reveillette/DCDC-website/pull/1.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/reveillette/DCDC-website/pull/1#event-269949733.

leidio committed:
Style

leidio committed:
Register window corrections

leidio committed:
Extensive register modal updates

leidio committed:
Submit spacing

leidio committed:
stupid space

leidio committed:
social buttons ul

leidio committed:
sorry, button experiments

because the register is a module i can’t see changes locally.  thus all
the commits when there shouldn’t be.

leidio committed:
Login changes

leidio committed:
new login/signup modal experiment

leidio committed:
again

leidio committed:
modal js path

leidio committed:
again

leidio committed:
again

leidio committed:
undo signup/signin modal experiment
